arch/x86/include/asm/extable.h

```diff
@@ -3,6 +3,7 @@
 #define _ASM_X86_EXTABLE_H
 
 #include <asm/extable_fixup_types.h>
+#include <asm/processor.h>	/* for cpu_relax() */
```

Comment:
I remember it was me who suggested to add it to the upstream header rather than just include it in pkvm files. But later I got some pushback in upstream about it: https://lore.kernel.org/all/20250418115710.4154311-1-dmaluka@chromium.org/T/ So it might be better to just add this include to pkvm files for simplicity. One patch less, and so on :)

Reply:
Sure, will merge this to the pkvm file.